addons/sourcemod/scripting/shavit-mapchooser.sp

```diff
@@ -931,7 +965,7 @@ void DoMapChangeAfterMapVote(char map[PLATFORM_MAX_PATH], char displayName[PLATF
 		CreateDataTimer(MapChangeDelay(), Timer_ChangeMap, data);
 		data.WriteString(map);
 		data.WriteString("RTV Mapvote");
-		// ClearRTV();
+		ClearRTV();
```

Comment:
this undid the previous commit

Reply:
> this undid the previous commit

also, seems that that commented out clearrtv actually breaks it and makes it get stuck on a map so i have removed the comment from it.

Reply:
I don't see the reason for this to be commented out. I saw it said probably fixed a bug with rtv after rtv but i cant see a scenario where that happens. if vote happens it either changes maps instantly or when rtv is typed after "nextmap" is set it instantly changes anyway.

Reply:
It makes the map change back to itself if you `rtv` between the time of the vote going through and before it actually switches.
```
[SM] zzz wants to rock the vote! Map vote will now start ...
[SM] Voting for next map has started.
[SM] Map voting has finished. The next map will be [T3] bhop_406. (Received 100% of 1 votes)
[SM] zzz wants to rock the vote! Map will now change to bhop_trigger_zones_test ...
```

>also, seems that that commented out clearrtv actually breaks it and makes it get stuck on a map so i have removed the comment from it.

I tested a bit and didn't encounter this so I'm assuming it's not a problem anymore